### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Benmuiruri**
Style the page to match Zeplin's design.

### CODE REVIEW

File: src/components/Book.module.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&family=Roboto+Slab:wght@100;300;500;600;700&display=swap');
+
+.bookDiv {
+  background-color: #fff;
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Book.module.css:12]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file. ıt is just an example if you have some repeated style please do the same thing.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.module.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+.author {
+  margin: 0.188rem 0 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.8rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
```
Comment (ahmetbozaci) [inline src/components/Book.module.css]: - [ ] Again for repeating the same rules for [DRY CSS](https://speckyboy.com/good-bad-css-practices/) please combine them.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @Benmuiruri  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/Book.module.css (hunk 1 above)
Comment (Benmuiruri) [reply src/components/Book.module.css:12]: Hi @ahmetbozaci thanks for that advice I will make that change.

File: src/components/Book.module.css (hunk 2 above)
Comment (Benmuiruri) [reply src/components/Book.module.css]: Hi @ahmetbozaci thanks for that advice I will  make that change

**Comment by Benmuiruri** [comment]
Thanks @ahmetbozaci for helping me see how I can make my code DRY.

File: src/components/AddBook.js [hunk 3]
```diff
@@ -30,30 +30,30 @@ const AddBook = () => {
...
               type="text"
               ref={titleInputRef}
-              placeholder="Title..."
+              className={styles.titleInput}
+              placeholder="Title"
             />
           </label>
-          <button type="submit">Add Book</button>
+          <button type="submit" className={styles.formBtn}>Add Book</button>
         </form>
```
Comment (prolajumokeoni) [inline src/components/AddBook.js:57]: ### OPTIONAL 

- [ ] Kindly align the form to have same width with the booklist container 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_141  |  Screenshot_140

**Review by prolajumokeoni — APPROVED** [review]
Hi @Benmuiruri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
-
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comment_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**